src/plugins/config/Config.tsx

```diff
@@ -77,6 +82,14 @@ const Config: FC = () => {
     <Formik initialValues={initialValues} onSubmit={handleSubmit}>
       <NoPaddingWrapper>
         <ResetForm initialValues={initialValues} />
+        <div>
```

Comment:
This div probably isn't necessary but not a big deal either way.

Reply:
Yeah it was a copy paste left over. I had originally changed it to return a div with the form and p as children but that made the entire config window 5px high. I can fix this.